pivotal-cron-resource.yml

```diff
@@ -0,0 +1,5 @@
+name: git
```

Comment:
the name should be `cron`. I have seen this pattern happening multiple times, looks like I need to change the documentation a bit.